packages/epo-widget-lib/src/widgets/CameraFilter/SpectrumDisplay/SpectrumDisplay.tsx

```diff
@@ -0,0 +1,110 @@
+import { between } from "@/lib/utils";
+import { FunctionComponent } from "react";
+import { Filter, spectrums } from "../data";
+import SpectrumLabels from "./SpectrumLabels";
+
+interface SpectrumDisplayProps {
+  min: number;
+  max: number;
+  range: number;
+  isCondensed: boolean;
+  activeFilter?: Filter;
+}
+
+const SpectrumDisplay: FunctionComponent<SpectrumDisplayProps> = ({
+  min,
+  max,
+  range,
+  isCondensed,
+  activeFilter,
+}) => {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      preserveAspectRatio="xMinYMin meet"
+      viewBox={`${min} 0 ${range} 200`}
+      data-testid="spectrum-display"
+    >
+      <defs>
+        {spectrums.map(({ name, stops }) => (
+          <linearGradient key={name} id={`${name}-gradient`}>
+            {stops.map(({ offset, stopColor, id }) => (
+              <stop
+                key={`${offset}-${stopColor}`}
+                {...{ offset, stopColor, id }}
+              />
+            ))}
+          </linearGradient>
+        ))}
+        <mask id="spectrumMask">
```

Comment:
this mask changes based on the active filter and just renders two transparencies on either side of the active area